download.dd

```diff
@@ -6,7 +6,54 @@ src="images/github-ribbon.png" alt="Fork D on GitHub"></a>
 
 $(D_S Downloads,
 
-$(H2 DMD $(DMDV2))
+$(TABLEC download-compilers,
+  $(T caption,
+    $(DIVID download-choose, ($(LINK2 http://wiki.dlang.org/Compilers, more information)))
+    $(H2 Choose a compiler)
+  )
+  $(TR
+    $(TD $(LINK2 #dmd                                         , $(IMG compiler-dmd.png)))
+    $(TD $(LINK2 http://gdcproject.org/downloads              , $(IMG compiler-gdc.svg)))
+    $(TD $(LINK2 http://github.com/ldc-developers/ldc/releases, $(IMG compiler-ldc.png)))
```

Comment:
Seems rather unfair that gdc gets the infinitely resizeable svg, and poor dmd and ldc are such with pngs.  But then again I'm not complaining. :)